extensions.json

```
@@ -88,6 +88,11 @@
       "version": "0.4.5",
       "checkout": "0.4.5"
     },
+    {
+      "id": "apollographql.apollo-midnight-color-theme",
+      "repository": "https://github.com/apollographql/apollo-midnight",
+      "version": "0.9.0"
```

No need to pin a specific version, since we're auto-publishing straight from the default branch.

```suggestion
      "repository": "https://github.com/apollographql/apollo-midnight"
```